src/driver/aurora-mysql/AuroraMysqlQueryRunner.ts

```diff
@@ -2411,14 +2411,24 @@ export class AuroraMysqlQueryRunner
      * Builds create table sql
      */
     protected createTableSql(table: Table, createForeignKeys?: boolean): Query {
-        const columnDefinitions = table.columns
+        const tableMetaData = this.connection.entityMetadatas.find(
```

Comment:
we should not rely on entityMetadatas in this method. Please send the `Table` instance without virtual columns into this method.

Reply:
I have updated this with the changes that you suggested. I was not aware that the table object was being used to be a direct reflection of the table in the database (it makes sense and it cleaned up those checks). Thanks @pleerock !